yarn.lock

```diff
@@ -2,13 +2,6 @@
 # yarn lockfile v1
 
```

Comment:
remove pls this file from this PR (it should be identical to master`s version).

Reply:
Vitalii see on the https://github.com/wahanegi/vibereport/pull/19/commits/4ffb00a9bc747b9ab7c832d5d908bde143b6a619
I put this file to the .gitignore but I think it is right